yt/frontends/gamer/data_structures.py

```diff
@@ -246,16 +246,20 @@ def _set_code_unit_attributes(self):
             setdefaultattr( self, 'mass_unit',   self.quan(self.parameters['Unit_M'], 'g' ) )
             setdefaultattr( self, 'time_unit',   self.quan(self.parameters['Unit_T'], 's' ) )
 
+            if self.mhd:
```

Comment:
Can we get this somehow from `parameters` instead of `self.mhd`?

Reply:
It is just an alias of `self. parameters["Magnetohydrodynamics"]`. See https://github.com/yt-project/yt/pull/2306/files#diff-0b29b3dd04114f5c2572cf5015f9a505R323